core/src/main/java/io/questdb/cutlass/Services.java

```diff
@@ -53,6 +54,7 @@ protected Services() {
     @Nullable
     public HttpServer createHttpServer(
             HttpServerConfiguration configuration,
+            PublicPassthroughConfiguration publicPassthroughConfiguration,
```

Comment:
There is a tiny bit of vermicelli here. There is top-level ServerConfiguratio, which includes all others. It feels we better pass that thru, rather than adding more method args

Reply:
addressed in https://github.com/questdb/questdb/pull/4479/commits/260070f7419b2cbc11ddb44cc19a75f80c49195c